packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 46.9.0 - August 8, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Process errors not in JSON format to make them user friendly before displaying ([PR #1521](https://github.com/forcedotcom/salesforcedx-vscode/pull/1521))
```

Comment:
This PR was released in 46.8.0 so we can remove it.

Reply:
will do.